dts/bindings/usb/usb-bc12.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for USB BC1.2 devices
+
+description: Common fields for USB BC1.2 devices
+compatible: "usb-bc12"
```

Comment:
No need for a description and compatible for a template binding file, this is not meant to be used on a node directly right? May just leave it empty.

Reply:
Done.